src/framework/vst/internal/vstaudioclient.h

```diff
@@ -83,6 +83,7 @@ class VstAudioClient
     void addParamChange(const ParamChangeEvent& param);
 
     bool m_isActive = false;
+    bool m_pluginHandlesVolume = true;
```

Comment:
```suggestion
    bool m_pluginHandlesVolume = false;
```
(it is better to use false as an uninitialized default value)

Reply:
Done.